.gitignore

```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+
+/package-lock.json
```

Comment:
Please remove the empty space in this file to resolve git error

Reply:
I have removed the spaces but which git error?